Events/src/main/java/com/github/swcraftlyon/meetup/infra/dao/MangoEventDao.java

```diff
@@ -0,0 +1,41 @@
+package com.github.swcraftlyon.meetup.infra.dao;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+
+public class MangoEventDao implements IEventDAO {
+
+    public static final String EVENT_DB = "EventDB";
+    public static final String MONGODB_LOCALHOST_27017 = "mongodb://localhost:27017";
```

Comment:
when you will have a conf _file_ you probably won't expose it to you application anyway, I'll abstract it into a config _class_. So you can put your hardcoded url into a config class right now, it will be easy to evolve configuration system later.